contracts/contracts/token/WOETH.sol

```diff
@@ -31,6 +43,25 @@ contract WOETH is ERC4626, Governable, Initializable {
         OETH(address(asset())).rebaseOptIn();
     }
 
+    function initialize2() external onlyGovernor {
+        if (_oethCreditsInitialized) {
+            require(false, "Initialize2 already called");
+        }
```

Comment:
Why not just `require(!_oethCreditsInitialized, "Initialize2 already called");`?

Reply:
good point: https://github.com/OriginProtocol/origin-dollar/pull/2106/commits/057469c08067ff67e31b8e1f524b61e249787499